build.rs

```diff
@@ -1,17 +0,0 @@
-use std::fs;
-use std::process::Command;
-
-fn main() {
-    // Update git submodule
-    Command::new("git")
```

Comment:
> Should we remain this git operation in build.rs script? I didn't see anyother place do this work.

I think any operation related to state of source code should be done by hand, `git submodule update --init` is not really a step of build. Developer should clone by `git clone <url> --recursive` rather than rely on cargo to do it for them.

Reply:
You've got a point.